tests/Form/Extension/Field/Type/FormTypeFieldExtensionTest.php

```diff
@@ -72,7 +72,7 @@ public function testbuildViewWithNoSonataAdminArray(): void
     //        $admin = $this->createMock('Sonata\AdminBundle\Admin\AdminInterface');
     //        $admin->expects($this->once())->method('getCode')->will($this->returnValue('admin_code'));
     //
-    //        $fieldDescription = $this->createMock('Sonata\AdminBundle\Admin\FieldDescriptionInterface');
+    //        $fieldDescription = $this->createMock('Sonata\AdminBundle\FieldDescription\FieldDescriptionInterface');
```

Comment:
Do you know why this test is commented ?

Reply:
No idea, I've created https://github.com/sonata-project/SonataAdminBundle/pull/6981